src/npc.c

```diff
@@ -358,7 +358,45 @@ INCLUDE_ASM(s32, "npc", npc_do_player_collision, Npc* npc);
 
 INCLUDE_ASM(void, "npc", npc_do_gravity, Npc* npc);
 
-INCLUDE_ASM(s32, "npc", func_800397E8);
+s32 func_800397E8(Npc* npc, f32 arg1) {
+    if (!(npc->flags & 0x208))
+    {
```

Comment:
We don't use the allman brace style here - could you put this on the same line as the if?